docs/self-managed/troubleshooting/platform-deployment.md

```diff
@@ -0,0 +1,17 @@
+---
+id: platform-deployment
+title: "Platform deployment"
+---
+
+## Connection issues between Identity and Keycloak
+
+By default, Keycloak requires all non-local IP ranges to use SSL. It has been noted that in certain
+cloud environments where communication between containers is performed using IPs outside the local range,
+components are not able to function correctly.
```

Comment:
I would add an example of the logged exception (there should be some examples in the JIRA issues), this way you can get to this page just by googling the error message